packages/openneuro-app/src/scripts/users/user-account-view.tsx

```diff
@@ -114,9 +129,8 @@ export const UserAccountView: React.FC<UserAccountViewProps> = ({
         setRows={handleLinksChange}
         className="custom-class"
         heading="Links"
-        // eslint-disable-next-line no-useless-escape
-        validation={/((([A-Za-z]{3,9}:(?:\/\/)?)(?:[-;:&=\+\$,\w]+@)?[A-Za-z0-9.-]+|(?:www.|[-;:&=\+\$,\w]+@)[A-Za-z0-9.-]+)((?:\/[\+~%\/.\w-_]*)?\??(?:[-\+=&;%@.\w_]*)#?(?:[\w]*))?)/} // URL validation regex
-        validationMessage="Invalid URL format. Please use a valid link."
+        validation={httpHttpsRequiredUrlValidation}
```

Comment:
Might be better to just check if `new URL()` fails for a string to avoid a regexp for it.

Reply:
cool, that is way better. I have it added.